front-end/src/app/app.component.ts

```diff
@@ -86,56 +85,64 @@ export class AppComponent {
   doneArray: Task[] = [];
   isDoneArrayEmpty = true;
   shouldShowTaskArray = false;
-  id = -1;
 
   applyForm = new FormGroup({
-    title: new FormControl(''),
-    comment: new FormControl(''),
+    title: new FormControl(null),
```

Comment:
you define the default values to the form control here. The default form values should never be null, it would be much better as an empty string. I know, the form reset will set the control values to null right now, but [that can be fixed as well](https://stackoverflow.com/a/75889986)